quint/test/types/inferrer.test.ts

```diff
@@ -56,21 +68,24 @@ describe('inferTypes', () => {
     assert.isEmpty(errors, `Should find no errors, found: ${[...errors.values()].map(errorTreeToString)}`)
 
     const stringTypes = Array.from(types.entries()).map(([id, type]) => [id, typeSchemeToString(type)])
+    // _printUpdatedStringTypes(stringTypes)
     assert.sameDeepMembers(stringTypes, [
+      [7n, '(bool) => int'],
+      [8n, 'bool'],
+      [9n, 'bool'],
+      [10n, 'int'],
+      [11n, 'bool'],
+      [12n, 'bool'],
+      [13n, 'int'],
+      [14n, 'int'],
+      [15n, '((bool) => int, bool) => int'],
+      [16n, '((bool) => int, bool) => int'],
       [1n, '∀ t0, t1 . (t0) => t1'],
       [2n, '∀ t0 . t0'],
       [3n, '∀ t0 . t0'],
       [4n, '∀ t0 . t0'],
       [5n, '∀ t0, t1 . ((t0) => t1, t0) => t1'],
-      [6n, '(bool) => int'],
-      [7n, 'bool'],
-      [8n, 'bool'],
-      [9n, 'int'],
-      [10n, 'bool'],
-      [11n, 'bool'],
-      [12n, 'int'],
-      [13n, 'int'],
-      [14n, '((bool) => int, bool) => int'],
+      [6n, '∀ t0, t1 . ((t0) => t1, t0) => t1'],
```

Comment:
These ids have a weird order (7-16, then 1-6). I guess if you feel like fixing, nice, otherwise its fine as well, this is not important.

Reply:
I will. Or mess with it so long as the tests are passing. I wonder why the order is that way!